Makefile

```diff
@@ -36,4 +37,10 @@ tests:
 	tox
 
 bench: install-dev
-	$(VENV)/bin/loads-runner --config=./loadtests/bench.ini --server-url=$(SERVER_URL) loadtests.TestPOC.test_all
+	$(VENV)/bin/loads-runner --config=./loadtests/bench.ini --server-url=$(SERVER_URL) loadtests.TestBasic.test_all
+
+loadtests-check:
+	make serve & PID=$$!; \
```

Comment:
maybe we could put make server on its own line, so in case it fails to start we don't run the load tests.